### ISSUE TITLE
pronto 10.X compatibility

### ISSUE THREAD

**Issue by oreststetsiak** [grodowski/pronto-undercover#3]
Hi @grodowski !
[pronto](https://rubygems.org/gems/pronto) has released recently, and changed major version to **10.X**
we actually switched and using it now
and we can not start using [pronto-undercover](https://github.com/grodowski/pronto-undercover) now, because dependency in latest **0.1.3** version:
the issue we have:
```
ostet@daisy:~/coding/overhaul-backend$bundle install
Warning: the running version of Bundler (1.17.2) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Fetching gem metadata from https://rubygems.org/.......
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies........
Bundler could not find compatible versions for gem "pronto":
  In snapshot (Gemfile.lock):
    pronto (= 0.10.0)

  In Gemfile:
    pronto-brakeman was resolved to 0.10.0, which depends on
      pronto (~> 0.10.0)

    pronto-undercover was resolved to 0.1.3, which depends on
      pronto (~> 0.9.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

would be nice to have **pronto-undercover** compatible with latest **pronto**
thanks in advance for any help
have a nice day!

**Comment by al3rez** [comment]
This PR would fix this problem #5

**Comment by oreststetsiak** [comment]
works fine, thanks a lot @azbshiri  @grodowski
